docs/en/install-upgrade/air-gapped-install.asciidoc

```diff
@@ -0,0 +1,532 @@
+
+A custom container would need to be created following similar instructions to setting up a web server in the <<air-gapped-elastic-artifact-registry,self-managed install case>>. For example, a container file using an NGINX base image could be used to run a build similar to the example described in <<air-gapped-elastic-artifact-registry-example>>.
+
+[discrete]
+[[air-gapped-k8s-os-elastic-endpoint-artifact-repository]]
+==== 2.4. Elastic Endpoint Artifact Repository
+
+Just like the {artifact-registry}. A custom container needs to be created following similar instructions to setting up a web server for the <<air-gapped-elastic-artifact-registry,self-managed install case>>.
+
+[discrete]
+[[air-gapped-k8s-os-ironbank-secure-images]]
+==== 2.5. Ironbank Secure Images for Elastic
+
+Besides the public link:https://www.docker.elastic.co[Elastic container repository], most {stack} container images are also available in Platform One's link:https://ironbank.dso.mil/repomap?vendorFilters=Elastic&page=1&sort=1[Iron Bank].
+
+[discrete]
+[[air-gapped-elastic-package-registry-example]]
+=== Appendix A - {package-registry}
+
+The following script generates a SystemD service file on a RHEL 8 system in order to run EPR with Podman in a production environment.
+
+[source,shell]
+----
+#!/usr/bin/env bash
+
+EPR_BIND_ADDRESS="0.0.0.0"
+EPR_BIND_PORT="8443"
+EPR_TLS_CERT="/etc/elastic/epr/epr.pem"
+EPR_TLS_KEY="/etc/elastic/epr/epr-key.pem"
+EPR_IMAGE="docker.elastic.co/package-registry/distribution:8.4.3"
+
+podman create \
+  --name "elastic-epr" \
+  -p "$EPR_BIND_ADDRESS:$EPR_BIND_PORT:$EPR_BIND_PORT" \
+  -v "$EPR_TLS_CERT:/etc/ssl/epr.crt:ro" \
+  -v "$EPR_TLS_KEY:/etc/ssl/epr.key:ro" \
+  -e "EPR_ADDRESS=0.0.0.0:$EPR_BIND_PORT" \
+  -e "EPR_TLS_CERT=/etc/ssl/epr.crt" \
+  -e "EPR_TLS_KEY=/etc/ssl/epr.key" \
+  "$EPR_IMAGE"
+
+## creates service file in the root directory
+# podman generate systemd --new --files --name elastic-epr --restart-policy always
+----
+
+The following is an example of an actual SystemD service file for an EPR, launched as a Podman service.
+
+[source,shell]
+----
+# container-elastic-epr.service
+# autogenerated by Podman 4.1.1
+# Wed Oct 19 13:12:33 UTC 2022
+
+[Unit]
+Description=Podman container-elastic-epr.service
+Documentation=man:podman-generate-systemd(1)
+Wants=network-online.target
+After=network-online.target
+RequiresMountsFor=%t/containers
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+Restart=always
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStart=/usr/bin/podman run \
+	--cidfile=%t/%n.ctr-id \
+	--cgroups=no-conmon \
+	--rm \
+	--sdnotify=conmon \
+	-d \
+	--replace \
+	--name elastic-epr \
+	-p 0.0.0.0:8443:8443 \
+	-v /etc/elastic/epr/epr.pem:/etc/ssl/epr.crt:ro \
+	-v /etc/elastic/epr/epr-key.pem:/etc/ssl/epr.key:ro \
+	-e EPR_ADDRESS=0.0.0.0:8443 \
+	-e EPR_TLS_CERT=/etc/ssl/epr.crt \
+	-e EPR_TLS_KEY=/etc/ssl/epr.key docker.elastic.co/package-registry/distribution:8.6.2
```

Comment:
This contains a fixed version number. Can we make this to the same one as the page you are viewing?

Reply:
Fixed in [40dbeb3](https://github.com/elastic/stack-docs/pull/2458/commits/40dbeb3c10d7aa1d5314b996b9c13f4a0acf5e9c)